src/setup/devnet/index.js

```diff
@@ -404,13 +404,58 @@ export class Devnet {
                         `${ganacheUser}@${ganacheURL.hostname}:~/data`
                     ], {stdio: getRemoteStdio()})
 
-                    // Run ganache in tmux
-                    await execa('ssh', [
+                    // Generate service files
+                    for (let i = 0; i < this.totalNodes; i++) {
+                        await execa('scp', [
+                            `-o`, `StrictHostKeyChecking=no`, `-o`, `UserKnownHostsFile=/dev/null`,
+                            `-i`, `~/cert.pem`,
+                            `${this.config.targetDirectory}/service.sh`,
+                            `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}:/home/${this.config.devnetBorUsers[i]}/`
+                        ], {stdio: getRemoteStdio()})
+
+                        if (i == 0) {
+                            await execa('ssh', [
+                                `-o`, `StrictHostKeyChecking=no`, `-o`, `UserKnownHostsFile=/dev/null`,
+                                `-i`, `~/cert.pem`,
+                                `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}`,
+                                `bash ${this.config.targetDirectory}/service-host.sh`
+                            ], {stdio: getRemoteStdio()})
+
+                            // TODO: Target location would vary depending on bor/heimdall version
```

Comment:
Can you remove the `TODO`?
Maybe `NOTE` is enough as there is no action from devs at this point in time.
Also please specify that current setup works with `0.3.x`

Reply:
Modified: [51d4a88](https://github.com/maticnetwork/matic-cli/pull/91/commits/51d4a88ac9e7c36db6384a2fe3b6a8a5a1d96199)